source/fundamentals/linq.txt

```diff
@@ -805,6 +805,9 @@ The following are some methods supported by the {+driver-long+} implementation o
 
    * - ``Distinct``
      - Returns distinct documents that match the specified criteria
+   
+   * - ``Exists``
+     - Tests whether a field exists
```

Comment:
S: would phrase these more in terms of what the method returns like some of the other methods:
```suggestion
   
   * - ``Exists``
     - Returns ``true`` if a field exists and false if it does not.
```